python/ray/tune/suggest/suggestion.py

```diff
@@ -251,6 +254,11 @@ def create_trial_if_possible(self, experiment_spec, output_path):
         logger.debug("creating trial")
         trial_id = Trial.generate_id()
         suggested_config = self.searcher.suggest(trial_id)
+        if suggested_config == Searcher.FINISHED:
```

Comment:
document `Searcher.searcher` ret type as dict|str|None and/or specify what returning FINISHED actually does

Reply:
done